pkgs/tools/misc/tmux/default.nix

```diff
@@ -1,10 +1,11 @@
 { stdenv, fetchurl, ncurses, libevent, pkgconfig }:
 
 stdenv.mkDerivation rec {
-  name = "tmux-2.0";
+  name = "tmux-${version}";
+  version = "2.0";
 
   src = fetchurl {
```

Comment:
Also `fetchFromGitHub` won't work with the release urls, as it fetches URLs of the form `https://github.com/${owner}/${repo}/archive/${rev}.tar.gz`. Maybe a `fetchGitHubRelease` could be useful (~70 uses in nixpkgs).